docs/release/role-handbooks/release-lead/README.md

```diff
@@ -168,6 +168,11 @@ The goal of this task is to ensure we are always using the latest Go version for
 
 Note: If the Go minor version of one of our supported branches goes out of supported, we should consider bumping
 to a newer Go minor version according to our [backport policy](./../../../../CONTRIBUTING.md#backporting-a-patch).
+Usually this is done after k/k bumps the Go version in their release branches. For example:
+* Cluster API v1.10 uses Kubernetes v1.32: https://github.com/kubernetes-sigs/cluster-api/blob/v1.10.0/go.mod
```

Comment:
May be
```suggestion
* Cluster API v1.10 has Kubernetes v1.32 as a dependency, which originally was using Go 1.22: https://github.com/kubernetes-sigs/cluster-api/blob/v1.10.0/go.mod
```

Reply:
Added the first half the suggestion. Not sure if the latter is helpful. Similar explanation as above.

The whole text here starts with "If the Go minor version of one of our supported branches goes out of support, we should consider bumping"

And this listing here is to explain how to lookup the right Go version we should use (I'll change the text above the listing)